### ISSUE TITLE
Enhance API documentation

### ISSUE THREAD

**Issue by bochaco** [maidsafe/safe_app_nodejs#313]
We need to document the experimental APIs with descriptive details and example snipets for how to use them. It could be worthy trying to also enhance the format of the webpage generated for the docs as discussed in PR #312.

**Comment by bochaco** [comment]
If I have to suggest something I'd say that we should first try to enhance the content of the documentation we have for the APIs, and in parallel with a lower priority try to see which library is the best for presenting such a content in a website like we do today.
